diagrams/sequence-diagrams/filling-in-a-form.md

```diff
@@ -0,0 +1,52 @@
+# Filling in a form
+
+```mermaid 
+sequenceDiagram
+  actor user as Form filler
+  participant browser as Web Browser
+  participant govuk as GOV.UK website
+  participant runner as forms-runner
+  participant api as forms-api
+  participant notify as GOV.UK Notify
+  participant inbox as shared email inbox
+  actor processor as Form processor
+
+  user->>browser: Visit GOV.UK page<br />that links to the form
+  browser->>govuk: GET
+  govuk-->>browser: Show page
+
+  user->>browser: Click link to form
+  browser->>runner: GET /form/{form id}/{form slug}
+  note over runner: TODO session management
```

Comment:
Is this to do in the current PR, or to do later?

Reply:
I think we should do this later and improve these diagrams. It seems like a good starting point for us and we can improve as we go along